src/main/java/fi/hsl/transitdata/stopestimates/MetroEstimateStopEstimatesFactory.java

```diff
@@ -33,25 +32,27 @@
             return Collections.emptyList();
         }
 
-        return metroEstimate.getMetroRowsList().stream()
+        List<InternalMessages.StopEstimate> metroStopEstimates = metroEstimate.getMetroRowsList().stream()
                 .flatMap(metroStopEstimate -> {
                     final int stopSequence = metroEstimate.getMetroRowsList().indexOf(metroStopEstimate) + 1;
                     return toStopEstimates(metroEstimate, metroStopEstimate, stopSequence, timestamp).stream();
                 })
-                //TODO: if there is only a single stop estimate with SKIPPED status, assume that it is a technical problem with metro ATS and change it to SCHEDULED
                 .collect(Collectors.toList());
+
+        //If more than one stop has been cancelled for a single metro, assume that cancellations are valid
```

Comment:
@samijuhani @mjaakko If this is fine, the PR is fine

Reply:
Yes, this is inteded behavior: 
>nuo CANCELLED rivit tulee kahdella tavalla: joko liikenteenohjaaja syystä tai toisesta käyttää "siirry reitille" toimintoa aikataulunäkymistä tai sitten jonkin järjestelmävirheen takia junalle jää merkkaamatta asemallakäyntiä. Jälkimmäisessä tapauksessa välistä merkkaamattomat asemat merkitään CANCELLED tilaan.

Reply:
Yes, this is intended behavior:
>nuo CANCELLED rivit tulee kahdella tavalla: joko liikenteenohjaaja syystä tai toisesta käyttää "siirry reitille" toimintoa aikataulunäkymistä tai sitten jonkin järjestelmävirheen takia junalle jää merkkaamatta asemallakäyntiä. Jälkimmäisessä tapauksessa välistä merkkaamattomat asemat merkitään CANCELLED tilaan.